### PR TITLE
add export CSV with fiat values

### PR DESCRIPTION
**Pull request by talvasconcelos**

### CODE REVIEW

**Comment by talvasconcelos** [comment]
@dni , does this need `make bundle` ?

**Comment by dni** [comment]
yes it needs a bundle all fles that are included inside the package.json `vendor_js`

**Comment by talvasconcelos** [comment]
Closes #1666

File: lnbits/core/templates/core/wallet.html [hunk 1]
```diff
@@ -101,12 +101,25 @@ <h3 class="q-my-none">
...
+                      <q-item-section>
+                        <q-item-label
+                          >{%raw%}{{$t('export_csv')}}{%endraw%}</q-item-label
+                        >
+                      </q-item-section>
+                    </q-item>
+                    <q-item clickable v-close-popup @click="exportCSVFiat">
+                      <q-item-section>
+                        <q-item-label
+                          >{%raw%}{{$t('export_csv')}}
```
Comment (motorina0) [inline lnbits/core/templates/core/wallet.html:116]: - maybe an `export_csv_fiat` label should be created
- `"(fiat)"` cannot be considered universal

**Review by motorina0 — APPROVED** [review]
LGTM 
 - see comment regarding i18n
 - fix conflict

**Comment by arcbtc** [comment]
Can we put a separate pr in for adding fiat value?

**Comment by arcbtc** [comment]
Sorry, closing, as decided this https://github.com/lnbits/lnbits/pull/1789

**Pull request closed without merging**